public/data/KendallDoesCoding.json

```diff
@@ -22,18 +32,19 @@
     {
       "name": "Quiz",
       "url": "https://kendalldoescoding.gq/quiz",
-      "icon": "globe"
+      "icon": "link"
     },
     {
       "name": "Look at my subdomains/projects",
       "url": "https://kendalldoescoding.gq/subdomains",
       "icon": "globe"
-    }
+    },
   ],
   "milestones": [
     {
       "title": "Started Tech Journey",
-      "date": "2020"
-    }
+      "date": "2020",
+      "icon": "globe"
+    },
```

Comment:
Remove trailing comma

Reply:
I think this is fixed in [dc12210](https://github.com/EddieHubCommunity/LinkFree/pull/1067/commits/dc122101b7e90058974c89d6ef0ec751b00034a1) but waiting not sure

Reply:
Hi, you forgot to add `,` on line 31.

Reply:
Yup, I deleted that cause on the above review by @emmalearnscode it showed line 31 without the ',' for some reason

Reply:
Remove the trailing comma on line 48.